### DISCUSSION TITLE
Used semantic PR GitHub check is no longer maintained

### DISCUSSION THREAD

**Discussion by stijnmoreels** [arcus-azure/arcus#251] [Ideas]
We use the [Semantic PR GitHub Action](https://github.com/zeke/semantic-pull-requests) to validate our PR's, but this service is no longer maintained. As an alternative, it is suggested to use this instead: https://github.com/marketplace/semantic-prs

All in favor, raise your ✋ 😄 .
@pim-simons @fgheysels

**Comment by pim-simons** [comment]
✋🏻

**Reply by stijnmoreels** [reply]
Ok, installed and removed the deprecated one. 👍

**Reply by pim-simons** [reply]
🏆
